docs/wrappers/ledgerutil_preamble.md

```diff
@@ -1,21 +1,126 @@
+      "key":"e01e1c4304282cc5eda5d51c41785bbe49636fbf174514dbd4b98dc9b9ecf5da",
+      "hashed":true,
+      "snapshotrecord1":null,
+      "snapshotrecord2":{
+        "value":"0c52a3dbc7b322ff35728afdd691244cfc0fc9c4743c254b57059a2394e14daf",
+        "blockNum":1,
+        "txNum":0
+      }
+    },
+    {
+      "namespace":"_lifecycle$$h_implicit_org_Org1MSP",
+      "key":"e01e1c4304282cc5eda5d51c41795bbe49636fbf174514dbd4b98dc9b9ecf5da",
+      "hashed":true,
+      "snapshotrecord1":{
+        "value":"0c52a3dbc7b322ff35728afdd691244cfc0fc9c4743c254b57059a2394e14daf",
+        "blockNum":1,
+        "txNum":0
+      },
+      "snapshotrecord2":null
+    },
+    {
+      "namespace":"marbles",
+      "key":"marble1",
+      "hashed":false,
+      "snapshotrecord1":{
+        "value":"{\"docType\":\"marble\",\"name\":\"marble1\",\"color\":\"blue\",\"size\":35,\"owner\":\"tom\"}"
+        "blockNum":3,
+        "txNum":0
+      },
+      "snapshotrecord2":{
+        "value":"{\"docType\":\"marble\",\"name\":\"marble1\",\"color\":\"blue\",\"size\":35,\"owner\":\"jerry\"}","blockNum":4,
+        "txNum":0
+      }
+    }
+  ]
+}
+```
+
+The field `ledgerid` indicates the name of the channel where the comparison took place. The field `diffRecords` provides a list of key/value differences found between the compared snapshots. The compare tool found 3 differences. The first diffRecord has a null value for the field `snapshotrecord1` indicating that the key is present in the second snapshot and missing in the first snapshot. The second diffRecord has a null value for the field `snapshotrecord2` indiciating the opposite scenario. Both diffRecords have a `hashed` value of true, indicating that they are private key/value differences. The third diffRecord contains data in both snapshot fields, indicating that the snapshots have divergent data for the same key. Further examination reveals the snapshots are in disagreement about the owner of the key `marbles marble1` and the height at which that value is set. The `hashed` field is set to false indicating this is a public key/value difference.
+
+## identifytxs
+
+The `ledgerutil identifytxs` command allows administrators to identify lists of transactions that have written to a set of keys in a peer's local block store. The command takes a JSON file input containing a list of keys and outputs a JSON file per key, each file containing a list of transactions, if the available block range is valid. When troubleshooting a divergence, the command is most effective when paired with the `ledgerutil compare` command, by taking a generated output JSON file as input and mapping the relevant keys to their respective transactions. The command does not necessarily have to be used in tandem with the `ledgerutil compare` command and can be used with a user generated JSON list of keys or an edited `ledgerutil compare` output JSON file to filter transactions in a more general approach.
```

Comment:
```suggestion
The `ledgerutil identifytxs` command allows administrators to identify lists of transactions that have written to a set of keys in a peer's local block store. The command takes a JSON file input containing a list of keys and outputs a JSON file per key, each file containing a list of transactions, if the available block range is valid. When troubleshooting a divergence, the command is most effective when paired with the `ledgerutil compare` command, by taking a `ledgerutil compare` generated output JSON file of key differences as input, so that `ledgerutil identifytxs` can identify the transactions on the blockchain associated with the in doubt keys. The command does not necessarily have to be used in tandem with the `ledgerutil compare` command and can be used with a user generated JSON list of keys or an edited `ledgerutil compare` output JSON file to filter transactions in a more general approach.
```